web/app/js/components/HealthPane.jsx

```diff
@@ -5,11 +5,11 @@ import React from 'react';
 import { Col, Progress, Row } from 'antd';
 import './../../css/health-pane.css';
 
-const TrafficIndicator = () => {
+const TrafficIndicator = props => {
```

Comment:
we can use the magiks of es6 and extract named parameters here:
`const TrafficIndicator = ({animate}) => {`
then we just need `animate` below

Reply:
amazing didn't know you could do this